vue3-apollo-quasar/src/composables/github/types/userRepos.ts

```diff
@@ -9,5 +9,5 @@ export interface Repo {
   isFork: boolean;
   stargazerCount: number;
   forkCount: number;
-  updatedAt: any;
+  updatedAt: string | null;
```

Comment:
is this supposed to be `string | null` or `string | undefined`? In the latter case, that would mean you do `updatedAt ?: string`

Reply:
null. not undefined